sdk/samples/core/mosquitto_rpc_server_sample.c

```diff
@@ -423,10 +424,10 @@ int main(int argc, char* argv[])
   // clean-up functions shown for completeness
   LOG_AND_EXIT_IF_FAILED(az_mqtt5_connection_close(&mqtt_connection));
 
-  if (mqtt5._internal.mosquitto_handle != NULL)
+  if (*mqtt5._internal.mosquitto_handle != NULL)
```

Comment:
can these be now changed to mosq since we have the reference to the handle in the application?